general/management/commands/create_front_page_caches.py

```diff
@@ -33,10 +35,15 @@ def handle(self, **options):
         logger.info("Updating front page caches")
 
         rss_url = settings.FREESOUND_RSS
-        pledgie_campaign = settings.PLEDGIE_CAMPAIGN
-        
+        donations_goal = settings.DONATIONS_GOAL
+        donations_date = settings.DONATIONS_DATE
+
         rss_cache = render_to_string('rss_cache.html', locals())
         cache.set("rss_cache", rss_cache, 2592000) # 30 days cache
 
-        pledgie_cache = render_to_string('pledgie_cache.html', locals())
-        cache.set("pledgie_cache", pledgie_cache, 2592000) # 30 days cache
\ No newline at end of file
+        donations = accounts.models.Donation.objects\
+                .filter(created__gt=donations_date).all().aggregate(Sum('amount'))
```

Comment:
This doesn't make a lot of sense to me. All donations which were done after the date of the donation? So we'll never be able to look back? Wouldn't it make more sense to create a Model for DonationCampaigns somehow and attach people to a particular campaign? That way we could also create new campaigns in the Admin rather than having to refresh the front page cache.
Also, careful: paypal lets people donate in various currencies, which could mean that you are summing over multiple currencies (EUR/$/...) which will make this break.